src/zoom_type.h

```diff
@@ -12,41 +12,41 @@
 	ZOOM_LVL_BEGIN  = 0, ///< Begin for iteration.
-	ZOOM_LVL_NORMAL = 0, ///< The normal zoom level.
+	ZOOM_LVL_IN_4X  = 0, ///< Zoomed 4 times in.
+	ZOOM_LVL_IN_2X,      ///< Zoomed 2 times in.
+	ZOOM_LVL_NORMAL,     ///< The normal zoom level.
 	ZOOM_LVL_OUT_2X,     ///< Zoomed 2 times out.
 	ZOOM_LVL_OUT_4X,     ///< Zoomed 4 times out.
 	ZOOM_LVL_OUT_8X,     ///< Zoomed 8 times out.
-	ZOOM_LVL_OUT_16X,    ///< Zoomed 16 times out.
-	ZOOM_LVL_OUT_32X,    ///< Zoomed 32 times out.
 	ZOOM_LVL_END,        ///< End for iteration.
 
 	/* Here we define in which zoom viewports are */
-	ZOOM_LVL_VIEWPORT = ZOOM_LVL_OUT_4X, ///< Default zoom level for viewports.
-	ZOOM_LVL_NEWS     = ZOOM_LVL_OUT_4X, ///< Default zoom level for the news messages.
-	ZOOM_LVL_INDUSTRY = ZOOM_LVL_OUT_8X, ///< Default zoom level for the industry view.
-	ZOOM_LVL_TOWN     = ZOOM_LVL_OUT_4X, ///< Default zoom level for the town view.
-	ZOOM_LVL_AIRCRAFT = ZOOM_LVL_OUT_4X, ///< Default zoom level for the aircraft view.
-	ZOOM_LVL_SHIP     = ZOOM_LVL_OUT_4X, ///< Default zoom level for the ship view.
-	ZOOM_LVL_TRAIN    = ZOOM_LVL_OUT_4X, ///< Default zoom level for the train view.
-	ZOOM_LVL_ROADVEH  = ZOOM_LVL_OUT_4X, ///< Default zoom level for the road vehicle view.
-	ZOOM_LVL_WORLD_SCREENSHOT = ZOOM_LVL_OUT_4X, ///< Default zoom level for the world screen shot.
-
-	ZOOM_LVL_DETAIL   = ZOOM_LVL_OUT_8X, ///< All zoomlevels below or equal to this, will result in details on the screen, like road-work, ...
-
-	ZOOM_LVL_MIN      = ZOOM_LVL_NORMAL, ///< Minimum zoom level.
-	ZOOM_LVL_MAX      = ZOOM_LVL_OUT_32X, ///< Maximum zoom level.
-
+	ZOOM_LVL_VIEWPORT = ZOOM_LVL_NORMAL, ///< Default zoom level for viewports.
+	ZOOM_LVL_NEWS     = ZOOM_LVL_NORMAL, ///< Default zoom level for the news messages.
+	ZOOM_LVL_INDUSTRY = ZOOM_LVL_OUT_2X, ///< Default zoom level for the industry view.
+	ZOOM_LVL_TOWN     = ZOOM_LVL_NORMAL, ///< Default zoom level for the town view.
+	ZOOM_LVL_AIRCRAFT = ZOOM_LVL_NORMAL, ///< Default zoom level for the aircraft view.
+	ZOOM_LVL_SHIP     = ZOOM_LVL_NORMAL, ///< Default zoom level for the ship view.
+	ZOOM_LVL_TRAIN    = ZOOM_LVL_NORMAL, ///< Default zoom level for the train view.
+	ZOOM_LVL_ROADVEH  = ZOOM_LVL_NORMAL, ///< Default zoom level for the road vehicle view.
+	ZOOM_LVL_WORLD_SCREENSHOT = ZOOM_LVL_NORMAL, ///< Default zoom level for the world screen shot.
+
+	ZOOM_LVL_DETAIL = ZOOM_LVL_OUT_2X, ///< All zoom levels below or equal to this will result in details on the screen, like road-work, ...
+	ZOOM_LVL_TEXT_EFFECT = ZOOM_LVL_OUT_2X, ///< All zoom levels above this will not show text effects.
+
+	ZOOM_LVL_MIN      = ZOOM_LVL_IN_4X, ///< Minimum zoom level.
+	ZOOM_LVL_MAX      = ZOOM_LVL_OUT_8X, ///< Maximum zoom level.
 };
 DECLARE_POSTFIX_INCREMENT(ZoomLevel)
 DECLARE_ENUM_AS_ADDABLE(ZoomLevel)
 
+static uint const ZOOM_BASE_SHIFT = static_cast<uint>(ZOOM_LVL_NORMAL);
+static uint const ZOOM_BASE  = 1U << ZOOM_BASE_SHIFT;
```

Comment:
Most nit-pickety nit-pick ever: Two spaces and yet no aligned =?

Reply:
Ah, that'll be due to renaming... oops.